gclient/tests/perf.rs

```diff
@@ -1,267 +0,0 @@
-use gclient::{GearApi, Result};
```

Comment:
This test had some important rationale, I guess. @shamilsan can you shed some light?

Reply:
Well the reasons i mentioned when asked for that test was completely different as I recall, this is rather useless now

Reply:
discussed.